quickwit/quickwit-search/Cargo.toml

```diff
@@ -33,9 +33,11 @@ tokio = { workspace = true }
 tokio-stream = { workspace = true }
 tokio-util = { workspace = true }
 tower = { workspace = true }
+ttl_cache = { workspace = true }
```

Comment:
Last released 4 years ago and is based on an also unmaintained crate. Let's keep that in mind.